### ISSUE TITLE
`InstanceRefresh` flagged as invalid value for `SuspendProcesses`

### ISSUE THREAD

**Issue by ghost** [aws-cloudformation/cfn-lint#2159]
*cfn-lint version: 0.54.3*

`InstanceRefresh` is missing from the allowed values in the `SuspendProcesses` key of the ASG rolling update policy:
```
E3016 Allowed values for SuspendProcesses are (Launch, Terminate, HealthCheck, ReplaceUnhealthy, AZRebalance, AlarmNotification, ScheduledActions, AddToLoadBalancer)
```

For reference, [the documentation](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_SuspendProcesses.html) lists the following:

- Launch
- Terminate
- AddToLoadBalancer
- AlarmNotification
- AZRebalance
- HealthCheck
- InstanceRefresh
- ReplaceUnhealthy
- ScheduledActions

Sample template:
```yaml
---
AWSTemplateFormatVersion: 2010-09-09
Resources:
  AutoScalingGroup:
    Type: AWS::AutoScaling::AutoScalingGroup
    UpdatePolicy:
      AutoScalingRollingUpdate:
        SuspendProcesses:
          - InstanceRefresh
    Properties:
      MinSize: 1
      MaxSize: 1
      LaunchConfigurationName: placeholder
```
